### ISSUE TITLE
SELECT with only one property returns empty array

### ISSUE THREAD

**Issue by s-h-a-d-o-w** [R-Vision/wmi-client#8]
Example:

`SELECT Name FROM Win32_Process`
Doesn't work. (returns an empty array)

`SELECT Name, ProcessId FROM Win32_Process`
Works.

**Comment by NAlexandrov** [comment]
Hello. I fixed this issue and also merged your PR #7 in develop branch. Could you test a new version in your environment and give me feedback? Thank you.

**Comment by s-h-a-d-o-w** [comment]
As for more general feedback (if that's what you meant):

I've done quite a bit of comparing of wmic projects as well as process list projects (since that's what I really needed) recently.
Numbers are looking pretty good for this project (this benchmark compared grabbing pids and ppids for a specific process name):
```
wmi-client x 8.98 ops/sec ±3.40% (47 runs sampled)
process-list x 14.56 ops/sec ±2.78% (70 runs sampled)
ps-node x 0.08 ops/sec ±0.99% (5 runs sampled)
tasklist x 6.48 ops/sec ±1.36% (35 runs sampled)
```
So... :+1: 

One thing I saw though - have you looked into whether you truly need that .xsl file? Because in one of the projects I looked at (might not be in this list), I think they used some parameter like /CSV or something over a custom .xsl. Which removes the hassle of having to provide the .xsl in the first place.

**Comment by NAlexandrov** [comment]
/CSV in Windows wmic is alias for %WINDIR%\System32\wbem\en-US\csv.xsl and I use this original csv.xsl with minimum modifications. So switching to /csv does not add speed.

process-list use native bindings to win api so it very fast.

**Comment by s-h-a-d-o-w** [comment]
Yes, that's why I wrote "the hassle of having to provide the .xsl in the first place".
It's not a matter of speed but of convenience.

But now that you're mentioning that it's just an alias, I remembered [this](https://stackoverflow.com/questions/9673057/wmic-error-invalid-xsl-format-in-windows7)...
I guess you avoid that by simply using your own file.
